Towards A Social Platform.md

```diff
@@ -0,0 +1,88 @@
+
+These would be use to fire up user base and generate buzz about the system; to demonstrate high value use of the system to the general public; allow research groups to “crowd source” solutions to their problems in some cases; and perhaps solve real problems! It would be great for us to come with a few seed competitions that User Engagement could run!
+### Resource Utilization and the Mileage model of resource utilization
+
+Assuming we are successful, we will ultimately need to deal with our limited resources. We could set across the board allocations but I would like to reward people for working in the system and sharing. So ultimately “frequent fliers”, “good samaritans”, and “wise-folk” should be allowed more resource as a reward for their utility to the system and its community. We could imagine a minimum allocation of space and some priority system on our execution managers. You can be promoted up to some maximum by a combination of frequent use, shared-objected used by others, and (maybe one day) by the number of times you have helped people outside your immediate environs (or something). But we should have a plan for this in some way,  
+### Newsfeeds and alerts
+
+Ideally, when a user logs in (or if they request alerts by email they should get them there) they should get an updates on things shared by people, groups, projects they follow and/or belong to; new public data about things they are interested in genes, organisms, environments, chemicals, pathways, etc.; new methods added to the system or updated. They might also be informed how many others have used their work product as well-- essentially being alerted when their metric change. 
+
+People could subscribe as well to various other newsfeeds perhaps we and others create: KBase events; KBase versions/data cycle; KBase Contests; KBase Science; etc. And one could imagine individuals groups and projects opening feeds for their members or the public. We might open a public “crowing” channel where KBase users can push their PR to the user base (pending our approval of course.)
+### Advertising results and “jobs” systemwide
+
+Following on that last thought we should enable our users to brag about what they done, ask for help, and offer jobs via our site. Our time-dependent aggregated metrics should also allow us to identify a hot topic in public space, pick it up and advertise it for that group (pending permission perhaps). 
+### Aggregating statistics
+
+We need to build relatively automated aggregators of the metrics above for both internal use and for public consumption. We must be mindful of privacy and side-effects of such metrics for projects and people and ensure that public metrics emphasize positive rather than show negative. 
+### Publication and DOI numbers
```

Comment:
Yes-- but there is also a social motivation part of this. We want people to get usage and citation metrics for their publications in system so that 1) there might be some positive feedback for them, 2) so others can see influential people, and 3) for a little bit of friendly competition.